asmcomp/cmmgen.ml

```diff
@@ -2825,10 +2825,16 @@ let transl_function f =
       Afl_instrument.instrument_function (transl env body)
     else
       transl env body in
+  let fun_codegen_options =
+    if !Clflags.optimize_for_speed then
+      [ Reduce_code_size ]
+    else
+      []
+  in
```

Comment:
It feels like you're reversing the logic.  "optimize for speed" means DO NOT reduce the code size.

Reply:
Sorry about that. Of course I reversed it. Thank you so much for noticing.